ssts/segmentation/segmentation_gmm.py

```diff
@@ -222,7 +222,7 @@ def get_windows(data, padding=3, flatten=True):
         n = 2 * padding + 1
         wins = np.zeros((h * w, n ** 2, c))
         for d in range(c):
-            X = np.pad(data[:, :, d], padding, "reflect")
+            X = np.pad(data[:, :, d], padding, "symmetric")
```

Comment:
@wesleyktatum  Before we were using "constant" or "reflective" which was adding constant values to the boundaries. Now I changed that to reflect the interior value on the exterior. This removes the frame artifact you were observing.